cmd/nodecmd/local.go

```diff
@@ -423,9 +429,44 @@ func localValidate(_ *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	rootDir := app.GetLocalDir(clusterName)
+	networkJSONPath := filepath.Join(rootDir, "network.json")
+	networkFile, err := os.ReadFile(networkJSONPath)
+	if err != nil {
+		return fmt.Errorf("could not read network config file of local cluster %s: %w", networkJSONPath, err)
+	}
+	var networkJSON anrnetwork.Config
+	if err = json.Unmarshal(networkFile, &networkJSON); err != nil {
+		return err
+	}
+	for _, node := range networkJSON.NodeConfigs {
```

Comment:
could you use cli.Status for the node listing? instead of loading network.json. it is standard way for ANR, later on to be replacing with tmpnet enabled version